README.md

```diff
@@ -0,0 +1,22 @@
+# This repository contains the catkin workspace for KNR Melson humanoid robot
+
+
+
+
+
```

Comment:
Trochę mało tego opisu    
Skoro to główne repo to warto się nieco rozpisać o samym projekcie melson.

Można też dać kilka linków, np do waszej podstrony na stronie koła: https://knr.edu.pl/melson/   
Na samym dole można też dopisać linki do innych repozytoriów melsonowych, które są jakoś powiązane.